bin/build-mock-web-services

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+# DOC: Build a new docker image for running the mock web services container and optionally push to Docker Hub if PUSH_IMAGE is set to anything. Builds for "linux/amd64" unless PLATFORM is set
+
+source bin/lib.sh
+
+IMAGE="mock-web-services"
+LOCATION="mock-web-services/"
+DOCKERFILE="mock-web-services/mock-web-services.Dockerfile"
+
+BUILD_ARGS="-f ${DOCKERFILE}
+  -t civiform/${IMAGE}:latest
+  --cache-from civiform/${IMAGE}
+  --build-arg BUILDKIT_INLINE_CACHE=1
+  --build-context server=server/
+  ${LOCATION}"
+
+PLATFORM_ARG=()
+if [[ "${PLATFORM}" ]]; then
```

Comment:
**nitpick:** https://google.github.io/styleguide/shellguide.html#testing-strings
```suggestion
if [[ -n "${PLATFORM}" ]]; then
```

Reply:
Same as the comment below. If I'm going to clean up these files I'd rather do them all together in a separate PR.